.github/workflows/build.yml

```diff
@@ -6,26 +6,36 @@ name: Build
 on:
   push:
     branches: [ master, development, '**-RC' ]
+  workflow_dispatch:
```

Comment:
Remove empty section

Reply:
thats only for testing purposes.
i'll remove that once all builds are tested